static/js/util/profile_edit.js

```diff
@@ -62,13 +63,17 @@ export function boundRadioGroupField(keySet: string[], label: string, options: O
         labelStyle={styles.labelStyle}
         value={obj.value}
         label={label}
+        aria-labelledby={labelId}
```

Comment:
material-ui is a whole ball of wax here...I'll wrap the `RadioButton` in a label tag, since when you pass the `label` prop in to `RadioButton` it doesn't seem to get properly connected to the input.